fdbclient/ServerKnobs.cpp

```diff
@@ -462,7 +462,15 @@ void ServerKnobs::initialize(Randomize _randomize, ClientKnobs* clientKnobs, IsS
 	init( REPLACE_INTERFACE_CHECK_DELAY,                         5.0 );
 	init( COORDINATOR_REGISTER_INTERVAL,                         5.0 );
 	init( CLIENT_REGISTER_INTERVAL,                            600.0 );
-	init( CLUSTER_CONTROLLER_ENABLE_WORKER_HEALTH_MONITOR,     false );
+	init( CC_ENABLE_WORKER_HEALTH_MONITOR,                     false );
+	init( CC_WORKER_HEALTH_CHECKING_INTERVAL,                   60.0 );
```

Comment:
Since some knobs are tunable, can we use BUGGIFY to verify:
it's safe to change the knobs to a different value. If we think a knob has a safe operation range, it's good to add a comment to guide SRE for knob tuning.